src/types.rs

```diff
@@ -348,9 +350,86 @@ impl<T> Line<T>
     }
 }
 
+#[derive(PartialEq, Clone, Debug)]
+pub enum WindingOrder {
+    Clockwise,
+    CounterClockwise,
+}
+
 #[derive(PartialEq, Clone, Debug, Serialize, Deserialize)]
 pub struct LineString<T>(pub Vec<Point<T>>) where T: Float;
 
+impl<T: Float> LineString<T> {
+
+    /// Returns the winding order of this line
+    pub fn winding_order(&self) -> WindingOrder {
+        let shoelace = twice_area(self);
+        if shoelace < T::zero() {
+            WindingOrder::Clockwise
+        } else if shoelace > T::zero() {
+            WindingOrder::CounterClockwise
+        } else if shoelace == T::zero() {
+            // what should be done here?
```

Comment:
I was considering making `WindingOrder` have 3 enum values: `Clockwise`, `CounterClockwise" and `Open` (as in 'this linestring is open, so there is no winding order'), since in theory this method can be called on every linestring, including linestrings which are not a closed ring.

However I didn't know about the other failure cases. So maybe a `None` would be better then.